### PR TITLE
Report repo

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1026

### Change description ###
ReportRepository, EnvelopeSummary added

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/data/model/EnvelopeSummary.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import java.time.Instant;
+
+public class EnvelopeSummary {
+
+    private final String container;
+    private final String fileName;
+    private final Instant fileCreatedAt;
+    private final Instant dispatchedAt;
+    private final Status status;
+    private final boolean isDeleted;
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/data/model/EnvelopeSummary.java]: public

File: src/main/java/uk/gov/hmcts/reform/blobrouter/data/ReportRepository.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    public ReportRepository(
+        NamedParameterJdbcTemplate jdbcTemplate,
+        EnvelopeSummaryMapper mapper
+    ) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.mapper = mapper;
+    }
+
+    public List<EnvelopeSummary> find(Instant from, Instant to) {
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/data/ReportRepository.java]: perhaps:

```suggestion
    public List<EnvelopeSummary> getEnvelopeSummary(Instant from, Instant to) {
```

?

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/data/ReportRepository.java]: do not mind

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/data/ReportRepository.java]: repository is now `ReportRepository` - should call names more specific than `find`. Imagine the usage:

```java

@Autowired
private ReportRepository reportRepository;

var findings = reportRepository->find(from, to);
```

🤷‍♂ find reports i guess 😄

**Pull request merged**